NineChronicles.Standalone/GraphTypes/ActionMutation.cs

```diff
@@ -0,0 +1,55 @@
+using GraphQL;
+using GraphQL.Types;
+using Libplanet;
+using Libplanet.Action;
+using Libplanet.Blockchain;
+using Libplanet.Crypto;
+using Nekoyume.Action;
+using Serilog;
+using System;
+using NineChroniclesActionType = Libplanet.Action.PolymorphicAction<Nekoyume.Action.ActionBase>;
+
+namespace NineChronicles.Standalone.GraphTypes
+{
+    public class ActionMutation : ObjectGraphType<NineChroniclesNodeService>
+    {
+        public ActionMutation()
+        {
+            Field<NonNullGraphType<BooleanGraphType>>("createAvatar",
+                resolve: context =>
+                {
+                    try
+                    {
+                        NineChroniclesNodeService service = context.Source;
+                        PrivateKey privatekey = service.PrivateKey;
+                        BlockChain<NineChroniclesActionType> blockChain = service.Swarm.BlockChain;
+                        Address userAddress = privatekey.PublicKey.ToAddress();
+                        Address avatarAddress = userAddress.Derive("avatar");
+
+                        var action = new CreateAvatar
+                        {
+                            avatarAddress = avatarAddress,
+                            index = 0,
+                            hair = 0,
+                            lens = 0,
+                            ear = 0,
+                            tail = 0,
+                            name = "createbymutation",
+                        };

```

Comment:
It seems you assumed this mutation must be used only for testing, right?

Reply:
Right. In this time, I only assumed using only for testing.